src/kw_include.sh

```diff
@@ -30,6 +30,9 @@ function include()
     return 0
   fi
 
-  declare -g "${varname}"=1
+  ret=$(declare -g "${varname}"=1 2>&1)
+  if [[ -n "$ret" ]]; then
+    return 1
```

Comment:
In `kw` we use the errno code for handling failures; in this sense, I think 1 is not the correct code here (EPERM 1 Operation not permitted)

Reply:
I was following the reference at the beginning of the file, but now I think I have a better idea for doing this!